Extend ADMIN SHOW DDL docs (#17378)
pingcap/docs: sql-statements/sql-statement-admin-show-ddl.md

```ebnf+diagram
AdminStmt ::=

>     'ADMIN' 'SHOW' 'DDL'

Reviewer: Do we need to specify the different meanings of three usages
```
"admin show ddl job queries limit m" -> "ADMIN SHOW DDL JOB QUERIES LIMIT 0, m"
"admin show ddl job queries limit n, m" -> "ADMIN SHOW DDL JOB QUERIES LIMIT n, m"
"admin show ddl job queries limit m offset n" -> "ADMIN SHOW DDL JOB QUERIES LIMIT n, m"
```

Author: I don't think that's needed. Most people will already be familiar with this because of SELECT statements

>     ( 
>         'JOBS' Int64Num? WhereClauseOptional 
>     |   'JOB' 'QUERIES' NumList 
>     |   'JOB' 'QUERIES' 'LIMIT' m 'OFFSET' n
>     )

Reviewer 2: ```suggestion
    ( 
        'JOBS' Int64Num? WhereClauseOptional 
    |   'JOB' 'QUERIES' NumList 
    |   'JOB' 'QUERIES' 'LIMIT' m ( ('OFFSET' | ',') n )?
    )
```

### `ADMIN SHOW DDL`

To view the status of the currently running DDL jobs, use `ADMIN SHOW DDL`. The output includes the current schema version, the DDL ID and address of the owner, the running DDL jobs and SQL statements, and the DDL ID of the current TiDB instance. The returned result fields are described as follows:

> - `SCHEMA_VER`: A number indicating the version of the schema.
> - `OWNER_ID`: The UUID of the DDL owner. See also [`TIDB_IS_DDL_OWNER()`](/functions-and-operators/tidb-functions.md).
> - `OWNER_ADDRESS`: The IP address of the DDL owner.
> - `RUNNING_JOBS`: Details about the running DDL job.
> - `SELF_ID`: The UUID of the TiDB node you are connected to. If this is the same as the `OWNER_ID` then you are connected to the DDL owner.
> - `QUERY`: The statement of the query.

Reviewer 3: ```suggestion
- `SCHEMA_VER`: a number indicating the version of the schema.
- `OWNER_ID`: the UUID of the DDL owner. See also [`TIDB_IS_DDL_OWNER()`](/functions-and-operators/tidb-functions.md).
- `OWNER_ADDRESS`: the IP address of the DDL owner.
- `RUNNING_JOBS`: details about the running DDL job.
- `SELF_ID`: the UUID of the TiDB node to which you are currently connected. If `SELF_ID` is the same as the `OWNER_ID`, it means that you are connected to the DDL owner.
- `QUERY`: the statement of the query.
```

```sql
ADMIN SHOW DDL\G;
```

```sql

> mysql> ADMIN SHOW DDL;

Reviewer 2: ```suggestion
mysql> ADMIN SHOW DDL \G
```

(or just remove this line)

> mysql> ADMIN SHOW DDL \G

Reviewer 3: ```suggestion
```

- `DB_NAME`: the name of the database where the DDL operation is performed.
- `TABLE_NAME`: the name of the table where the DDL operation is performed.
- `JOB_TYPE`: the type of DDL operation. Common job types include the following:
    - `create schema`: for [`CREATE SCHEMA`](/sql-statements/sql-statement-create-database.md) operations.
    - `create table`: for [`CREATE TABLE`](/sql-statements/sql-statement-create-table.md) operations.

>     - `create view`:for [`CREATE VIEW`](/sql-statements/sql-statement-create-view.md) operations.

Reviewer 3: ```suggestion
    - `create view`: for [`CREATE VIEW`](/sql-statements/sql-statement-create-view.md) operations.
```

>     - `ingest`: Ingestion with accelerated index backfilling as configured by [`tidb_ddl_enable_fast_reorg`](/system-variables.md#tidb_ddl_enable_fast_reorg-new-in-v630).
>     - `txn`: Basic transactional backfill.

Reviewer 3: ```suggestion
    - `ingest`: ingestion with accelerated index backfilling as configured by [`tidb_ddl_enable_fast_reorg`](/system-variables.md#tidb_ddl_enable_fast_reorg-new-in-v630).
    - `txn`: basic transactional backfill.
```

>     - `add index /* txn-merge */`: Transactional backfill with a temporary index that gets merged with the original index when the backfill is finished.

Reviewer 3: ```suggestion
    - `add index /* txn-merge */`: transactional backfill with a temporary index that gets merged with the original index when the backfill is finished.
```

>     - `txn`: basic transactional backfill.

Reviewer 3: ```suggestion
    - `txn`: basic transactional backfilling.
```

>     - `txn`: basic transactional backfill.
>     - `add index /* txn-merge */`: transactional backfill with a temporary index that gets merged with the original index when the backfill is finished.

Reviewer 3: ```suggestion
    - `txn`: basic transactional backfilling.
    - `add index /* txn-merge */`: transactional backfilling with a temporary index that gets merged with the original index when the backfilling is finished.
```

    - `delete only`, `write only`, `delete reorganization`, `write reorganization`: these four states are intermediate states. For their specific meanings, see [How the Online DDL Asynchronous Change Works in TiDB](/ddl-introduction.md#how-the-online-ddl-asynchronous-change-works-in-tidb). As the intermediate state conversion is fast, these states are generally not visible during operation. Only when performing `ADD INDEX` operation can the `write reorganization` state be seen, indicating that index data is being added.
    - `public`: indicates that it exists and is available to users. Generally, after `CREATE TABLE` and `ADD INDEX` (or `ADD COLUMN`) operations are completed, it will become the `public` state, indicating that the newly created table, column, and index can be read and written normally.
- `SCHEMA_ID`: the ID of the database where the DDL operation is performed.
- `TABLE_ID`: the ID of the table where the DDL operation is performed.
- `ROW_COUNT`: when performing the `ADD INDEX` operation, it is the number of data rows that have been added.

> - `CREATE_TIME`: the create time of the DDL operation.

Reviewer 3: ```suggestion
- `CREATE_TIME`: the creation time of the DDL operation.
```
